core-api/src/main/java/com/optimizely/ab/event/NoopEventHandler.java

```diff
@@ -29,7 +29,8 @@
     private static final Logger logger = LoggerFactory.getLogger(NoopEventHandler.class);
 
     @Override
-    public void dispatchEvent(String url, Map<String, String> params) {
-        logger.debug("Called dispatchEvent with URL: {} and params: {}", url, params);
+    public void dispatchEvent(LogEvent logEvent) {
```

Comment:
From my POV as a consumer of this API I don't see much anymore complexity in dealing with LogEvent or a raw String and map.  Auto complete makes it pretty easy to see that LogEvent has a method for getting the URL and getting the params. Down the road it might be useful to add more methods to it like one that returns a map of headers and header values. 

It also seems useful to put the RequestMethod on LogEvent.  I am not sure I would have known to use GET for the old event API if I didn't work at Optimizely.

Do we really need to the EventApiV1.. abstraction?  Wouldn't the only thing that ever changes be the endpoint and params?  These are internal to the EventApi class so changing those wouldn't break public API compatibility.